open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/configuration/properties/EngineHostServicesConfig.java

```diff
@@ -93,10 +129,9 @@ public void setEngineServiceConfigs(List<EngineServiceConfig> engineServiceConfi
     public String toString()
     {
         return "EngineHostServicesConfig{" +
-                       "engineServiceConfigs=" + engineServiceConfigs +
-                       ", OMAGServerPlatformRootURL='" + getOMAGServerPlatformRootURL() + '\'' +
-                       ", OMAGServerName='" + getOMAGServerName() + '\'' +
-                       '}';
+                "engineList=" + engineList +
+                ", engineServiceConfigs=" + engineServiceConfigs +
```

Comment:
if engineServiceConfigs is not set as it is optional, does this toString look ok?

Reply:
yes, this is ok